docker/projfs

```diff
@@ -86,6 +86,18 @@ def system(*args)
   end
 end
 
+# NOTE: when hosted on a Mac, our bind-mounted home directory likely does
+#       not support xattrs, but /tmp does, so use that instead
+def test_options
+  test_options=[]
```

Comment:
```suggestion
  test_options = []
```

Reply:
I suppose it might also be an anti-pattern to name a local variable the same as a method in the same scope.

Reply:
OK, will change -- thanks!